changelog/fragments/operator-name-config.yaml

```diff
@@ -0,0 +1,14 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Remove `--operator-name` from `generate` subcommands in favor of using `project-name`
```

Comment:
Nit, not a PR blocker: I ended up having to update these to be past tense `Removed` because the changelog read weird with just `Remove`. For the migration guide `Remove` makes sense since it's an instruction.